consumers/writers/influxdb/setup_test.go

```diff
@@ -24,7 +24,7 @@ func TestMain(m *testing.M) {
 		"INFLUXDB_USER_PASSWORD=test",
 		"INFLUXDB_DB=test",
 	}
-	container, err := pool.Run("influxdb", "1.6.4-alpine", cfg)
+	container, err := pool.Run("influxdb", "1.8.3", cfg)
```

Comment:
Why are we abandoning `alpine`? Goal is to keep all images as small as possible.